include/industry_standard/spdm.h

```diff
@@ -1001,29 +1001,42 @@ typedef struct {
 /* SPDM SET_CERTIFICATE request */
 typedef struct {
     spdm_message_header_t header;
-    /* param1 == BIT[0:3]=slot_id, BIT[4:7]=RSVD
-     * param2 == RSVD
-     * void * cert_chain*/
+    /* param1 == BIT[0:3]=slot_id, Request Attribute in 1.3
+     * param2 == key_pair_id in 1.3
+     * uint8_t cert_chain[]; */
 } spdm_set_certificate_request_t;
 
+#define SPDM_SET_CERTIFICATE_REQUEST_SLOT_ID_MASK 0xF
+
+/* SPDM SET_CERTIFICATE request Attributes */
```

Comment:
Can also define fields for `CSRCertModel` and `SetCertModel`.

Reply:
I don't define it because the spec said the value and format is same.

`CSRCertModel` - `This field indicates the desired certificate model of the CSR. The value and format of this field shall be the same as CertModel in Certificate info`
`SetCertModel` - `This field indicates the certificate model of the certificate chain. The value and format of this field shall be the same as CertModel in Certificate info.`